pkg/kratos/handlers.go

```diff
@@ -582,15 +603,33 @@ func (a *API) deleteKratosSession(w http.ResponseWriter) {
 	// This is hacky as it does not call the Kratos API and is likely to break on
 	// a new Kratos version, but there is no easy way to delete the session
 	// from the Kratos API
-	c := &http.Cookie{
+	c := deleteKratosCookie()
+	http.SetCookie(w, c)
+}
+
+func deleteKratosCookie() *http.Cookie {
```

Comment:
suggestion(non-blocking): Naming here is a little strange.
May I suggest refactoring this method to `kratosDeletionCookie` or something that reflects the fact that a cookie is returned? I'm saying this because looking at line 366 in service.go it looks misleading/counterintuitive